Twig/Extension/PostMetaExtension.php

```diff
@@ -49,6 +49,7 @@ public function getFunctions()
     {
         return array(
             new \Twig_SimpleFunction('wp_get_post_meta', array($this, 'getPostMeta')),
+            new \Twig_SimpleFunction('wp_get_image_url_from_id', array($this, 'getImageUrlFromId')),
```

Comment:
@t-n-y I think you've missed to update this file too? ;)

It should be as proposed by @plouc:

```php
new \Twig_SimpleFunction('wp_get_image_url_from_id', 'wp_get_attachment_url'),
```